sycl/plugins/unified_runtime/ur/adapters/hip/memory.hpp

```diff
@@ -131,13 +131,21 @@ struct ur_mem_handle_t_ {
     struct SurfaceMem {
       hipArray *Array;
       hipSurfaceObject_t SurfObj;
-      ur_mem_type_t ImageType;
+      ur_image_desc_t ImageInfo;
```

Comment:
Nit, this is called `ImageDesc` in the constructor args, should we keep it the same for consistency?